weekly/2024/week-36.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Week 36 in Packit
+date: 2024-09-09
+authors: lbarczio
+tags:
+  - 2024-September
+  - 2024
+  - September
+---
+
+## Week 36 (September 3rd – September 9th)
+
+- Packit now supports passing custom arguments to various static analyzers through `--csmock-args` CLI option and `csmock_args` configuration. ([packit#2402](https://github.com/packit/packit/pull/2402))
+- When syncing a new release, Packit is now able to fast forward a specified merge to a configured list of branches.
+  Use the new syntax for `dist_git_branches` as in this example to utilise this functionality:
```

Comment:
Seems a bit cryptic to me..;)
This is a long-awaited feature and I am not sure if people would guess it is that what they are waiting for..;-D


Something like this?

```suggestion
- When syncing a new release, Packit can now avoid diverging git branches by reusing the commit from one branch when opening pull requests into other branches. (These pull requests can be merged in fast-forward mode.)
  Use the new syntax for `dist_git_branches` as in this example to utilise this functionality:
```

Reply:
works for me, @majamassarini wdyt?